editors/code/src/commands/syntax_tree.ts

```diff
@@ -55,7 +55,7 @@ export function syntaxTree(ctx: Ctx): Cmd {
 
 // We need to order this after LS updates, but there's no API for that.
 // Hence, good old setTimeout.
-function afterLs(f: () => any) {
+function afterLs(f: () => unknown) {
```

Comment:
Hm, this maybe should be `=> void`, or whats the TypeScript alternative for `-> ()`?

Reply:
Also, this is just a horrible hack which needs to die in flames...

Reply:
Okay, let me revisit this)

Reply:
Yes, you are right, `void` is semantically `-> ()`, but not the unit type itself `()`. It is quite a special language item that is meant to be used only for the function return type and denotes that the value should be discarded (also function with `void` return type is covariant to other functions with different return type, in which way `void` type is different from `undefined` type). 

I would say that the concept of unit type the way it is present in Rust is not implemented in TypeScripts type system.